term-manager/impl/src/webapp/WEB-INF/components.xml

```diff
@@ -0,0 +1,53 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE beans PUBLIC "-//SPRING//DTD BEAN//EN" "http://www.springframework.org/dtd/spring-beans.dtd">
+
+<beans>	
+	<!-- Project logic   -->
+	<bean id="org.sakaiproject.acadtermmanage.logic.AcademicSessionLogic"
+    	class="org.sakaiproject.acadtermmanage.logic.impl.AcademicSessionLogicImpl"
+    	init-method="init">    	
+    	<property name="dao" ref="org.sakaiproject.acadtermmanage.dao.AcademicSessionDao" />
+  		<property name="as_sakaiProxy" ref="org.sakaiproject.acadtermmanage.logic.AcademicSessionSakaiProxy" />    		
+    	<property name="cache" ref="org.sakaiproject.acadtermmanage.sakai.academicsession_cache" />	    	
+    </bean>
+        
+    <!--  Project DAOs -->
+    <bean id="org.sakaiproject.acadtermmanage.dao.AcademicSessionDao"
+		class="org.sakaiproject.acadtermmanage.dao.impl.AcademicSessionDaoImpl"
+		init-method="init">		
+		<property name="cm_api" ref="org.sakaiproject.acadtermmanage.logic.AcademicSessionSakaiProxy" />
+	</bean>	    
+		
+	<!--  SakaiProxy -->
+ 	<bean id="org.sakaiproject.acadtermmanage.logic.AcademicSessionSakaiProxy"
+    	class="org.sakaiproject.acadtermmanage.logic.impl.AcademicSessionSakaiProxyImpl"
+        init-method="init">        
+        <property name="cmService" ref="org.sakaiproject.coursemanagement.api.CourseManagementService" />
+        <property name="cmAdmin" ref="org.sakaiproject.coursemanagement.api.CourseManagementAdministration" />  
+     	<property name="siteService" ref="org.sakaiproject.site.api.SiteService" />   
+     	<property name="eventService" ref="org.sakaiproject.event.api.EventTrackingService" />
+    	<property name="functionManager" ref="org.sakaiproject.authz.api.FunctionManager" />  
+    	<property name="serverConfigurationService" ref="org.sakaiproject.component.api.ServerConfigurationService" />
+        <property name="sessionManager" ref="org.sakaiproject.tool.api.SessionManager" />
+        <property name="securityService" ref="org.sakaiproject.authz.api.SecurityService" />
+        <property name="toolManager" ref="org.sakaiproject.tool.api.ToolManager" />    	
+        <property name="prefService" ref="org.sakaiproject.user.api.PreferencesService" /> 	
+	</bean>
+	
+	<!-- Cache: ttl=1 hour -->
+    <bean id="org.sakaiproject.acadtermmanage.sakai.academicsession_cache"
```

Comment:
Should use the Sakai's memory service for creating a cache vs using specifc EhCache implementation.